src/ontology/cl-edit.owl

```diff
@@ -27702,6 +27707,29 @@ SubClassOf(obo:CL_4023036 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534))
 SubClassOf(obo:CL_4023036 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_P32848))
 
 
+# Class: obo:CL_4023051 (vascular leptomeningeal like cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "ILX:0770143") obo:IAO_0000115 obo:CL_4023051 "A neurohypophyseal cell that expresses Pdgfra and Lum. They are fibroblast-esque cells that are attached to vessels, located in the space betwee smooth muscle and astrocytes. Vascular leptomeningeal cells (VLMCs) like cells are a family of functionally related (but organ-specific) mesothelial fibroblasts that form protective membranes around internal organs, including the pia and arachnoid membranes of the brain.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> obo:CL_4023051 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(oboInOwl:hasDbXref obo:CL_4023051 "ILX:0770143")
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4023051 "VLMC")
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4023051 "vascular leptomeningeal cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID: 30096314") rdfs:comment obo:CL_4023051 "The fact that ENMFBs were obtained by sorting Wnt1-Cre;R26Tomato cells indicates that these cells are derived from the neural crest, as has previously been shown for both pia and arachnoid.")
```

Comment:
This needs unpacking a bit if it is do go into CL + we would need the original reference.  
Presumably 'ENMFB' = this cell type?  If this is evidence for the NC origin of these cells then that should be stated up front with a bit more detail about the experiment being referred to.

It's hard for me to tell from the detail provided whether this statement would make more sense in the Brain Data Standards Ontology as  a detail of mapping single cell data to these cell types.